### ISSUE TITLE
Integrate bookmarks

### ISSUE THREAD

**Issue by tupton** [tupton/alfred-chrome-history#3]
Bookmarks are stored in a similar sqlite db; search both of these and integrate, like the Chrome omnibar.

This probably should include some sort of intelligent sorting; since we use `last_accessed_time` for history items, we need to figure out where to insert bookmark results.

**Comment by tupton** [comment]
> Bookmarks are stored in a similar sqlite db

Apparently it's a JSON file, but this can still be done. Not sure how useful it is for people like me who never bookmark anything in Chrome, but it could be useful for those with lots of bookmarks.

**Comment by Jyqft** [comment]
Would it be possible to implement this through another script filter? So that you won't have bookmark and history items mixing together.

**Comment by tupton** [comment]
> Would it be possible to implement this through another script filter?

Definitely. Not wanting to mix them is a big part of the reason that I haven't done this yet. There's already a [decent Chrome bookmarks workflow](http://www.packal.org/workflow/chrome-bookmarks-0); it seems that those who have and want to search through bookmarks should just use that. The initial goal I had with integrating bookmarks was to make this workflow emulate the omnibar, but I don't think that's really possible or desirable.

**Comment by Jyqft** [comment]
Oh, that's an ambitious and interesting idea. It sounds nice to mix them together if you put it that way.

The way Chrome handles omnibar search results has never been as good as, say, Firefox's, so emulating Chrome's way might not be the best bet. I'm not too sure how Firefox does it; I just remember that it always gave me more useful results when I used it back in the day (around version 2 to 4) than Chrome does right now.
